internal/pkg/cli/svc_init.go

```diff
@@ -253,6 +264,12 @@ func (o *initSvcOpts) Ask() error {
 	if err := o.validateSvc(); err != nil {
 		return err
 	}
+	if err := o.askIngressType(); err != nil {
+		return err
+	}
+	if err := o.validateIngressType(); err != nil {
+		return err
+	}
```

Comment:
should this call be in the `Validate()` method rather than `Ask`? I think after `askIngressType` it's guaranteed that the user selected a valid option from the prompt, but in `Validate` we check the flag value